api/v1/runtime/runtime_agent_api.proto

```diff
@@ -42,6 +42,8 @@ enum EventType {
   EVENT_PROCESS_OOM = 7;
   EVENT_MAGIC_WRITE = 8;
   EVENT_SIGNATURE = 9;
+
+  EVENT_ANY = 999;
```

Comment:
What's the difference between `ANY` and `UNKNOWN`?

Reply:
UNKNOWN is not a valid value. ANY is valid value used for any other event. Mostly useful in unit tests now.